app/javascript/locale/en.ts

```diff
@@ -31,6 +31,20 @@ export default {
       installed: "INSTALLED",
       statement: "Check your statement for this period's payout.",
       title: "Referral Stats",
+    },
+    welcome: {
+      description: "To get started on earning through the Brave Rewards system, here is a list of things we recommend setting up first.",
+      first: {
+        description: "Follow the instructions to verify that you're the rightful owner of properties.",
+        number: "1",
+        title: "Add your websites or channels",
+      },
+      second: {
+        description: "To receive payments, your wallet accounts must be connected. Connect Uphold for your desired payment options.",
+        number: "2",
+        title: "Connect wallet services",
+      },
+      title: "Weclome to Brave Rewards",
```

Comment:
*welcome

Reply:
Fixed the translations! Also added Japanese locales https://github.com/brave-intl/publishers/pull/2733/commits/b301a96874ed81adf0e72261f6d10b6df3c644f5